cmd/import.go

```diff
@@ -94,4 +106,8 @@ func runImportSql(absImportDir string) {
 	if err != nil {
 		log.Fatal().Err(err).Msg("error running the sql script")
 	}
+	_, err = runConfigFilesSync(absImportDir, xmlRpcUser, xmlRpcPassword)
+	if err != nil {
+		log.Fatal().Err(err).Msg("error synchronizing configuration files")
```

Comment:
Is this fatal error? IIUC when xmlrpc command fails that only means user should call that command manually to trigger config file refresh.
Shouldn't this be a regular error with proper explaining what to do next?

Reply:
I agree with you guys. Do you mean manually through our tool (which means a sub-command) or the spacecmd?

Reply:
I think for now we should use `spacecmd` command so we can merge this PR faster. Assumption is that we will not hit this error too often and we just need fallback.